e2e/app/page/workspaces-page.ts

```diff
@@ -178,4 +181,29 @@ export default class WorkspacesPage extends AuthenticatedPage {
     await waitWhileLoading(this.page);
     return selectMenu.getSelectedOption();
   }
+
+  async openAboutPage(workspaceCard: WorkspaceCard): Promise<WorkspaceAboutPage> {
+    await workspaceCard.clickWorkspaceName(false);
+    const aboutPage = new WorkspaceAboutPage(this.page);
+    // Older workspace requires Review Research Purpose
+    const reviewPurposeModal = new WorkspaceReviewResearchPurposeModal(this.page);
+    // Wait maximum 3 seconds to find Review Purpose modal.
+    const modalVisible = await reviewPurposeModal.isVisible(3000);
+    if (modalVisible) {
+      await reviewPurposeModal.clickReviewNowButton();
+      await aboutPage.waitForLoad();
+      // Click "Looks Good" link
+      await this.page
+        .waitForXPath('//a[text()="Looks Good"]', { visible: true, timeout: 2000 })
+        .then((link) => link.click())
+        .catch(() => {
+          // READER doesn't have permission to review research purpose. "Looks Good" link is not available to click.
+          // Ignore timeout error thrown by waitForXpath
```

Comment:
I guess in Puppeteer there's no other way built in to express 'doing this thing shouldn't work'?

Reply:
No. I could write a wrapper func but I don't have many usage at this time.